chroot-bin/runsommelier

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
I'm tempted to call this just `sommelier` (and then provide the full path to the actual sommelier executable inside).  Any opinions?

Reply:
commit 7fb307182c49aea6a8adddb09e7106bb0411491e